signatures/alert_signatures.py

```diff
@@ -0,0 +1,1012 @@
+    "ET WEB_SERVER disable_functions PHP config option in uri": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET WEB_SERVER open_basedir PHP config option in uri": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET WEB_SERVER auto_prepend_file PHP config option in uri": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET WEB_SERVER Access to /phppath/php Possible Plesk 0-day Exploit June 05 2013": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_SERVER PHP SERVER SuperGlobal in URI": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET SCAN NETWORK Incoming Masscan detected": MicroAttackStage.HOST_DISC,
+    "ET CURRENT_EVENTS Possible Magnitude IE EK Payload Nov 8 2013": MicroAttackStage.DATA_DELIVERY,
+    "ET WEB_SERVER Possible XXE SYSTEM ENTITY in POST BODY.": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET EXPLOIT Possible ZyXELs ZynOS Configuration Download Attempt (Contains Passwords)": MicroAttackStage.DATA_EXFILTRATION,
+    "ET TROJAN Miuref/Boaxxe Checkin": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET POLICY PE EXE or DLL Windows file download HTTP": MicroAttackStage.DATA_DELIVERY,
+    # "ET ATTACK_RESPONSE Output of id command from HTTP server": MicroAttackStage.INIT,
+    "ET WEB_SERVER Possible bash shell piped to dev tcp Inbound to WebServer": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET WEB_SERVER CURL Command Specifying Output in HTTP Headers": MicroAttackStage.DATA_DELIVERY,
+    "ET WEB_SERVER WGET Command Specifying Output in HTTP Headers": MicroAttackStage.DATA_DELIVERY,
+    "ET POLICY SSLv3 outbound connection from client vulnerable to POODLE attack": MicroAttackStage.NON_MALICIOUS,
+    "ET WEB_SERVER WEB-PHP phpinfo access": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_SERVER PHP.//Input in HTTP POST": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_SERVER PHP System Command in HTTP POST": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET CURRENT_EVENTS Unknown Malicious Second Stage Download URI Struct M2 Feb 06 2015": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET CURRENT_EVENTS Nuclear EK Landing Apr 03 2015": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET CURRENT_EVENTS Nuclear EK Landing Apr 08 2015": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET CURRENT_EVENTS Nuclear EK Landing Apr 22 2015": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET TROJAN Poweliks Clickfraud CnC M4": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET CURRENT_EVENTS Unknown Malicious Second Stage Download URI Struct Sept 15 2015": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_SERVER 401TRG Generic Webshell Request - POST with wget in body": MicroAttackStage.DATA_DELIVERY,
+    "ET TROJAN Possible NanoCore C2 60B": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET TROJAN Possible Metasploit Payload Common Construct Bind_API (from server)": MicroAttackStage.COMMAND_AND_CONTROL,
+    "GPL CHAT Jabber/Google Talk Outgoing Traffic": MicroAttackStage.NON_MALICIOUS,
```

Comment:
This alert signature ("GPL CHAT Jabber/Google Talk Outgoing Traffic") is `MicroAttackStage.NON_MALICIOUS`, whereas above it is `MicroAttackStage.INFO_DISC`. Which one to keep?